Polytope_distance_d/include/CGAL/Polytope_distance_d.h

```diff
@@ -21,6 +21,10 @@
 #ifndef CGAL_POLYTOPE_DISTANCE_D_H
 #define CGAL_POLYTOPE_DISTANCE_D_H
 
+#ifdef _MSC_VER
+# pragma warning(disable: 4244) // conversion warning in Boost iterator_adaptor
+#endif
```

Comment:
Same comment as previously: we cannot disable a warning without the `#pragma warning(push/pop)`.